.buildkite/Dockerfile

```diff
@@ -1,13 +1,18 @@
-FROM gcr.io/opensourcecoin/radicle-registry/ci-base
+FROM gcr.io/opensourcecoin/radicle-registry/ci-base@sha256:18d8cf1e39f4f42d871a2a9d3bcd8641bfda343a1fc7b9fbca4f8aea422bae6f
```

Comment:
Where does the hash come from?

Reply:
It's the latest image the registry provides.